cmd/parse/parse.go

```diff
@@ -31,26 +33,45 @@ func ParseCmd(cmdCfg *Config) *cobra.Command {
 	return &cobra.Command{
 		Use:     "parse",
 		Short:   "Start parsing the blockchain data",
-		PreRunE: types.ConcatCobraCmdFuncs(ReadConfig(cmdCfg), setupLogging),
+		PreRunE: ReadConfig(cmdCfg),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			parserData, err := SetupParsing(cmdCfg)
+			context, err := GetParsingContext(cmdCfg)
 			if err != nil {
 				return err
 			}
 
-			return StartParsing(parserData)
+			go StartPrometheus()
+
+			return StartParsing(context)
 		},
 	}
 }
 
+// StartPrometheus allows to start a Telemetry server used to expose useful metrics
+func StartPrometheus() {
+	cfg := types.Cfg.GetTelemetryConfig()
+	if !cfg.IsEnabled() {
+		return
+	}
+
+	router := mux.NewRouter()
+	router.Handle("/prometheus", promhttp.Handler())
```

Comment:
The convention is to use "/metrics" - that's the default in prometheus, it can be configured as something else but better to stick to the convention here